### ISSUE TITLE
Fliping board reset to start position

### ISSUE THREAD

**Issue by woprandi** [franciscoBSalgueiro/en-croissant#397]
### Describe the bug

When fliping board after (or during) position edition, it resets to initial  position

### Reproduction

1. Open analysis board
2. Edit a position
3. Flip the board
4. The position is reseted 

### Platform and versions

Version: 0.11.1

Tauri version: 1.6.1

OS: Windows_NT x86_64 10.0.19045

**Comment by jez9999** [comment]
Yeah, I get this too.  Rather annoying how many editing bugs there have been lol
